lighthouse-cli/test/fixtures/perf/frame-metrics.html

```diff
@@ -0,0 +1,27 @@
+<html>
+  <head>
+  </head>
+  <body>
+    <iframe id="frame" src="frame-metrics-inner.html" width="100%" height="700"></iframe>
+    <div id="space"></div>
+    <div id="content"></div>
+    <script>
+      const space = document.getElementById('space');
+      const content = document.getElementById('content');
+
+      // The innerHTML will be the LCP and FCP of the main frame.
+      // It is painted after the inner frame FCP, and it is smaller than the inner frame LCP.
+      // The metrics for all frames will therefore be different than the metrics for the main frame.
+      setTimeout(() => {
+        content.innerHTML = '<span>This is the main frame LCP and FCP.</span>';
+
+        // Add a small layout shift in the main frame.
+        // More layout shifts occur in the inner frame.
+        // CLS for all frames will therefore, be different than CLS for main frame.
+        setTimeout(() => {
+          space.style.height = '100';
+        }, 50);
+      }, 500);
```

Comment:
Tried to set this as far out as possible, but anything much larger than 500ms gets a `NO_TTI_CPU_IDLE_PERIOD` error.

Reply:
Add a 
```js
const keepAliveInterval = setInterval(() => stall(51), 1000)
setTimeout(() => clearInterval(keepAliveInterval), 10000)
setTimeout(() => doTheMainFrameFcp(), 8000)
```

we can go over it later today if you have time?

Reply:
I would expect 500 to be extremely flaky in CI :)